nirc_ehr/resources/queries/dbo/q_blood.sql

```diff
@@ -1,9 +1,15 @@
 
-SELECT anm.ANIMAL_ID_NUMBER AS Id,
-       CAST(anmEvt.EVENT_DATETIME AS TIMESTAMP) AS bloodDate,
+SELECT anmEvt.ANIMAL_EVENT_ID as objectid,
+       anm.ANIMAL_ID_NUMBER AS Id,
+       CAST(COALESCE (adt.CHANGE_DATETIME,anmEvt.EVENT_DATETIME) AS TIMESTAMP) AS bloodDate,
```

Comment:
Is adt.CHANGE_DATETIME the time at which the entry was last edited? If so, we shouldn't use it as the date of the blood draw. I expect we'd want to only use anmEvt.EVENT_DATETIME for bloodDate

Reply:
Yes, agree. Updated.